src/firmware/app/primitives/primitive_manager.c

```diff
@@ -139,11 +139,22 @@ void app_primitive_manager_startNewPrimitive(PrimitiveManager_t *manager,
                                              unsigned int primitive_index,
                                              const primitive_params_t *params)
 {
-    assert(primitive_index < PRIMITIVE_COUNT);
+    // We have both a assert statement and a if statement because we want to easily
```

Comment:
Should be "an assert", not "a assert". Also "an if..."

Reply:
changed!